.github/workflows/ci.yml

```diff
@@ -19,13 +19,13 @@ jobs:
       - name: Install dependencies
         run: npm install
       - name: Lint files
-        run: node Makefile.js lint
+        run: npm run lint
   test:
     name: Test
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node: [15.x, 14.x, 13.x, 12.x, 10.x, "10.12.0"]
+        node: [15.x, 14.x, 13.x, 12.x]
```

Comment:
We'll support Node 10 until the next ESLint major release, and we'll maybe need a new Acorn/Espree version before that.

Can we run only `unit:cjs` on Node 10?

Reply:
> Can we run only unit:cjs on Node 10?

No idea. Open for suggested changes though.

Reply:
separate job I guess? I dunno if there's a more elegant way.

Reply:
Separate job or conditionals (we have an example with `if` in eslint/eslint repo [here](https://github.com/eslint/eslint/blob/master/.github/workflows/ci.yml#L43-L46)).